operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/DefaultEvent.java

```diff
@@ -0,0 +1,16 @@
+package io.javaoperatorsdk.operator.processing.event;
+
+import io.fabric8.kubernetes.client.CustomResource;
+import java.util.function.Predicate;
+
+@SuppressWarnings("rawtypes")
+public class DefaultEvent extends AbstractEvent {
```

Comment:
Do we really need this class?

Reply:
We don't but I found a little bit annoying that AbstractEvent is declared abstract but in fact it does not have any abstract method so if you want to create a generic event, you'd need an anonymous class.

Reply:
removed

Reply:
I'll refactor `AbstractEvent` separately.